bug-1671522-pref-assess-change-to-windows-mouse-wheel-scroll-anima-release-82-83.toml

```diff
@@ -0,0 +1,32 @@
+[experiment]
+end_date = '2020-11-17'

```

Comment:
Also this normally isn't necessary unless you're overriding Experimenter, so you can cut it since we're making other changes here

```suggestion
```